### PR TITLE
ADEN-4183 Animate VUAP on Mercury.

### PR DESCRIPTION
**Pull request by drets**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [bfaaMobile.js#L13](https://github.com/Wikia/app/blob/2dd93968220c777034644d8d34db19c56dae0db6/extensions/wikia/AdEngine/js/template/bfaaMobile.js#L13): Function has 10 parameters which is greater than 7 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

File: extensions/wikia/AdEngine/js/template/bfaaMobile.js [hunk 1]
```diff
@@ -54,10 +55,39 @@ define('ext.wikia.adEngine.template.bfaaMobile', [
 		adsModule.setSiteHeadOffset(height);
 	}
 
+	function animate(startAspectRatio, finalAspectRatio) {
+		var windowWidth = document.body.clientWidth;
+
+		adSlot.style.height = windowWidth / startAspectRatio + 'px';
+
+		setTimeout(function () {
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/template/bfaaMobile.js:66]: I wonder if we can get any cleaner solutions than `setTimeout(() => {}, 0)`. What other options have you considered?

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [bfaaDesktop.js#L13](https://github.com/Wikia/app/blob/b00755beb33ed47e8f3d806465e099d0870ddfbf/extensions/wikia/AdEngine/js/template/bfaaDesktop.js#L13): Function has 10 parameters which is greater than 7 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. MAJOR [bfaaMobile.js#L13](https://github.com/Wikia/app/blob/b00755beb33ed47e8f3d806465e099d0870ddfbf/extensions/wikia/AdEngine/js/template/bfaaMobile.js#L13): Function has 10 parameters which is greater than 7 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

File: skins/oasis/css/core/ads.scss [hunk 2]
```diff
@@ -66,6 +66,7 @@ $taboola-small-breakpoint: "#{$screen} and (max-width: 420px)";
 	}
 
 	#HOME_TOP_LEADERBOARD, #TOP_LEADERBOARD, #CORP_TOP_LEADERBOARD, #HUB_TOP_LEADERBOARD {
+		background: #000;
```
Comment (nandy-andy) [inline skins/oasis/css/core/ads.scss]: Use defined [variable](https://github.com/Wikia/app/blob/9f536f72bfa49bbdf7f7563ef20a86aa09bdb56a/skins/shared/color.scss#L2)s, please.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [bfaaDesktop.js#L13](https://github.com/Wikia/app/blob/7e30c03965d1825ae035654cc1615164bdf92254/extensions/wikia/AdEngine/js/template/bfaaDesktop.js#L13): Function has 10 parameters which is greater than 7 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
2. MAJOR [bfaaMobile.js#L13](https://github.com/Wikia/app/blob/7e30c03965d1825ae035654cc1615164bdf92254/extensions/wikia/AdEngine/js/template/bfaaMobile.js#L13): Function has 10 parameters which is greater than 7 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

**Review by bognix — APPROVED** [review]

**Pull request merged**